Source/Core/RequestScheduler.js

```diff
@@ -0,0 +1,383 @@
+/*global define*/
+define([
+        './clone',
+        './Check',
+        './defined',
+        './defineProperties',
+        './Heap',
+        './isBlobUri',
+        './isDataUri',
+        './RequestState',
+        '../ThirdParty/Uri',
+        '../ThirdParty/when'
+    ], function(
+        clone,
+        Check,
+        defined,
+        defineProperties,
+        Heap,
+        isBlobUri,
+        isDataUri,
+        RequestState,
+        Uri,
+        when) {
+    'use strict';
+
+    /**
+     * Tracks the number of active requests and prioritizes incoming requests.
+     *
+     * @exports RequestScheduler
+     *
+     * @private
```

Comment:
So this is private now?  Don't we need a way to disable throttling and configure other parameters?